packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -574,3 +577,27 @@ class _ReorderableListContentState extends State<_ReorderableListContent> with T
     });
   }
 }
+
+/// A [GlobalKey] that uses a scope and a value to determine equality.
+///
+/// The scope is compared using [identical], while the value is compared
+/// using [operator ==]. This allows a locally scoped value to be turned
+/// into a globally unique key.
+class _ScopedValueKey extends GlobalKey {
```

Comment:
Let's try and put the word "Global" into this class name somewhere just for clarity.

Reply:
Done